dynamic-layers/meta-openembedded/networkmanager/files/networkmanager.initd

```diff
@@ -0,0 +1,17 @@
+#!/sbin/openrc-run
+# Copyright (c) 2008 Saleem Abdulrasool <compnerd@compnerd.org>
+# Distributed under the terms of the GNU General Purpose License v2
+# $Header: $
+
+supervisor=supervise-daemon
+command=/usr/sbin/NetworkManager
+command_args_foreground="-n"
+
+description="Network Manager Daemon"
+
+depend() {
+   need dbus
+   provide net
+}
+
+# vim: set ft=gentoo-init-d ts=3 sw=3 et:
```

Comment:
Keep the setting of the file type but drop the rest of this.  Along with that, reindent with either 4 spaces or tabs if necessary (I'm on mobile and can't tell).